TwilioVerifySDKTests/TwilioVerify/Sources/Domain/Factor/Mocks/PushFactoryMock.swift

```diff
@@ -27,8 +27,8 @@ class PushFactoryMock {
 }
 
 extension PushFactoryMock: PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String,
-                    serviceSid: String, identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func createFactor(withAccessToken accessToken: String, friendlyName: String, serviceSid: String, identity: String,
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/9cbd540c6135af3d1641ad17aef5ccc5e04d146c/TwilioVerifySDKTests/TwilioVerify/Sources/Domain/Factor/Mocks/PushFactoryMock.swift#L30"></span>Function should have 7 parameters or less: it currently has 8<br />
<code>function_parameter_count</code> <code>PushFactoryMock.swift:30</code></td>
    </tr>
  </tbody>
</table>